packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -265,7 +265,7 @@ function ETHBalance({ on, prefix = '' }: { on: NetworkType; prefix?: string }) {
   }
 
   return (
-    <span className="text-xl font-light text-white">
+    <span className="break-all text-xl font-light text-white">
```

Comment:
When eth/token amount number is very large, the string would overflow behind the transfer panel.

Part of the string would be cut off making it fail tests + bad UX.

Now when the string is very long it will wrap itself vertically if it runs out of space.